website/templates/project/modal_remove_contributor.mako

```diff
@@ -0,0 +1,122 @@
+                            <div class="panel panel-default">
+                                <div class="panel-body">
+                                    <div class="form-group" data-bind="if:contributorToRemove">
+                                        <span><b>{{contributorToRemove()["fullname"]}} will be</b> removed from the following projects and/or components.</span>
+                                    </div>
+                                    <div class="col-md-8" align="left">
+                                        <ul data-bind="foreach: { data: titlesToRemove(), as: 'item' }">
+                                            <li>
+                                                <h4 class="f-w-lg" data-bind="text: item"></h4>
+                                            </li>
+                                        </ul>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div data-bind="visible:titlesToKeep().length">
+                            <div class="panel panel-default">
+                                <div class="panel-body">
+                                    <div class="form-group" data-bind="if:contributorToRemove">
+                                        <span><b>{{contributorToRemove()["fullname"]}} cannot</b> be removed from the following projects and/or components.</span>
+                                    </div>
+                                    <div class="col-md-8" align="left">
+                                        <ul data-bind="foreach: { data: titlesToKeep(), as: 'item' }">
+                                            <li>
+                                                <h4 class="f-w-lg" data-bind="text: item"></h4>
+                                            </li>
+                                        </ul>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div><!-- end removeAll page -->
+                </div>
+                <div data-bind="if: !canRemoveNode() && !pageChanged()">
+                    <span><b>{{contributorToRemove()["fullname"]}}</b> cannot be removed as a contributor.  You need at least one administrator, bibliographic contributor, and one registered user.</span>
+                </div>
+                <div data-bind="if: pageChanged()">
+                    <span>Please save or discard your existing changes before removing a contributor.</span>
+                </div>
+            </div>
+            <!-- end modal-body -->
+
+            <div class="modal-footer">
+                <div data-bind="if:canRemoveNode() && !pageChanged()" align="right">
+                        <span data-bind="if: page() === REMOVE">
+                                <div class="row">
```

Comment:
Extra indent on this line and previous, ya? Also below in the `REMOVE_ALL` branch.